openreferee_server/operations.py

```diff
@@ -87,3 +94,32 @@ def cleanup_event(event):
     session = setup_requests_session(event.token)
     cleanup_event_tags(session, event)
     cleanup_file_types(session, event)
+
+
+def process_editable_files(session, files, endpoints):
+    uploaded = defaultdict(list)
+    for _file in files:
```

Comment:
`file` is no longer a builtin, and in any case it was super useless before (so pretty fine to shadow). In any case, it means you don't need the underscore here (btw, to avoid collisions you usually put it at the end, not at the beginning since that's for private/internal)